### PR TITLE
Allow empty, non-system topics to be deleted

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: temba/api/v2/views.py [hunk 1]
```diff
@@ -3457,6 +3457,9 @@ class TopicsEndpoint(ListAPIMixin, WriteAPIMixin, BaseEndpoint):
     write_serializer_class = TopicWriteSerializer
     pagination_class = CreatedOnCursorPagination
 
+    def get_queryset(self):
+        return super().get_queryset().filter(is_active=True)
```
Comment (ericnewcomer) [inline temba/api/v2/views.py:3461]: API shouldn't return deleted topics

File: temba/tickets/views.py [hunk 2]
```diff
@@ -85,7 +107,9 @@ class Form(forms.ModelForm):
             def __init__(self, **kwargs):
                 super().__init__(**kwargs)
 
-                self.fields["topic"].queryset = self.instance.org.topics.order_by("name")
+                self.fields["topic"].queryset = self.instance.org.topics.filter(is_active=True).order_by(
+                    "-is_system", "name"
+                )
```
Comment (ericnewcomer) [inline temba/tickets/views.py:112]: We should show system topics at the top

**Comment by norkans7** [comment]
We should now allow creating a new topic with the same name as a deleted one
https://github.com/nyaruka/rapidpro/pull/5415/files#diff-462b848ead19bdcf6fb248d3fe558d3778c5fbf620cbe1683d030173158b7aecL53

**Comment by ericnewcomer** [comment]
> We should now allow creating a new topic with the same name as a deleted one https://github.com/nyaruka/rapidpro/pull/5415/files#diff-462b848ead19bdcf6fb248d3fe558d3778c5fbf620cbe1683d030173158b7aecL53

Deleted topics are renamed with a uuid, so I don't think this matters. We could add a is_active filter here, but again, don't think it's necessary.

**Review by norkans7 — APPROVED** [review]

File: temba/tickets/tests.py [hunk 3]
```diff
@@ -275,6 +275,24 @@ def test_update(self):
...
+        response = self.assertDeleteFetch(delete_url, [self.editor, self.admin])
+        self.assertContains(response, "You are about to delete")
+
+        # submit to delete it
+        response = self.assertDeleteSubmit(delete_url, self.admin, object_deactivated=user_topic, success_status=302)
+
+        # we should have been redirected to the system topic
+        self.assertEqual(f"/ticket/{system_topic.uuid}/open/", response.url)
+
+        # system topics can't be deleted
```
Comment (rowanseymour) [inline temba/tickets/tests.py:294]: something to go here?

File: templates/tickets/topic_delete.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{% extends 'includes/modax.html' %}
+{% load i18n %}
+
+{% block fields %}
+    {% if has_tickets %}
```
Comment (rowanseymour) [inline templates/tickets/topic_delete.html:5]: 2 spaces

**Pull request merged**